8.0/debian/buster/developer/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM php:8.0-buster
+MAINTAINER "Bence Gazder"
+LABEL maintainer="bence.gazder@apicore.de"
+
+ENV COMPOSER_VERSION 2.0.11
```

Comment:
We now have 2.0.12.

The same applies to the other images as well.

Reply:
Set it for 7.4 and 8.0 versions